### PR TITLE
Add Contributors Section

### PR DESCRIPTION
**Pull request by gaeun**
I've added a "contributors section" to the GCI website showing the profile pictures, names, and number of commits of all the contributors to this repo.

### CODE REVIEW

**Comment by hvardhanx** [comment]
@gaeun Thanks for the pull request. Can you post the link to the website (via your github repo) for easier review?

File: index.html [hunk 1]
```diff
@@ -29,9 +30,16 @@
...
+			<h3>Contributors Section</h3>
+  		    <ul>
+		    </ul>
+		  </div>
+
+
         </section>
 
-        <aside id="sidebar">
+		        <aside id="sidebar">
```
Comment (robbyoconnor) [inline index.html]: Why is all of this trailing whitespace here?

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
See comments.

**Comment by gaeun** [comment]
@royalharsh https://github.com/gaeun/gci16.fossasia.org
@robbyoconnor Thanks, I didn't notice that. Just fixed it!

**Comment by hvardhanx** [comment]
Please squash your commits into one.

**Comment by hvardhanx** [comment]
This should help: [Squash your latest commits into one](https://davidwalsh.name/squash-commits-git)

**Comment by gaeun** [comment]
My git log doesn't show the last three commits I made, and only shows my first pull request. I'm not really sure why--is it because I edited the html file directly on Github?

**Comment by hvardhanx** [comment]
It might be possible because your local repo doesn't contain those changes you made directly on github. Try doing a `git pull` from your repo.

**Comment by daminisatya** [comment]
@gaeun Great job, just try to resolve your conflicts. and it's good to go! :)

**Comment by robbyoconnor** [comment]
Merge conflict resolution is a bit of a black art for newbies..

**Comment by gaeun** [comment]
@royalharsh Ah finally fixed it :) I squashed all my commits.
@daminisatya I'll try and figure that out by tonight. Thanks!

**Comment by daminisatya** [comment]
@gaeun Great job, Any doubt regarding resolving the conflict. You can contact any of the mentor.

**Comment by daminisatya** [comment]
Please watch this[ video](https://www.youtube.com/watch?v=g8BRcB9NLp4) to clear the merge conflicts.

**Comment by gaeun** [comment]
@daminisatya I think I've resolved all the conflicts. Thanks for the video link! That was helpful.

**Comment by daminisatya** [comment]
Amazing, any final review from your side @robbyoconnor ?

**Comment by hvardhanx** [comment]
Great job and congrats on your first pull request! 😄

**Pull request merged**